apps/bfd-pipeline/bfd-pipeline-rda-grpc/pom.xml

```diff
@@ -180,6 +180,20 @@
             <version>${mockito.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <!-- API, java.xml.bind module -->
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
+        </dependency>
+        <dependency>
+            <!-- Runtime, com.sun.xml.bind module -->
+            <groupId>org.glassfish.jaxb</groupId>
+            <artifactId>jaxb-runtime</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.annotation</groupId>
+            <artifactId>jakarta.annotation-api</artifactId>
```

Comment:
I'm curious why these needed to be added to the pom.  Is it to prevent older transitive dependencies from being brought in?